src/models/UserModel.js

```diff
@@ -0,0 +1,32 @@
+import mongoose from 'mongoose'
+import validator from 'validator'
+
+
+const {Schema, model} = mongoose
+const { isEmail } = validator 
+
+const userSchema = new Schema (
+    {
+        firstName: {
+            type: String,
+            required: true
+        },
+        lastName: {
+            type: String,
+            required: true
+        },
+        email: {
+            type: String,
+            required: true,
+            lowercase: true,
+            validate: [isEmail, 'Enter a valide email']
```

Comment:
Codacy has a fix for the issue: [Replace `············validate:·[isEmail,·'Enter·a·valide·email']` with `······validate:·[isEmail,·'Enter·a·valide·email'],`](https://app.codacy.com/gh/Denniman/group-four-BE/pullRequest?prid=7801432)
```suggestion
      validate: [isEmail, 'Enter a valide email'],
```